platform/android/export/export_plugin.cpp

```diff
@@ -590,9 +590,9 @@ zip_fileinfo EditorExportPlatformAndroid::get_zip_fileinfo() {
 
 Vector<String> EditorExportPlatformAndroid::get_abis() {
 	Vector<String> abis;
-	abis.push_back("armeabi-v7a");
-	abis.push_back("arm64-v8a");
-	abis.push_back("x86");
+	abis.push_back("arm32");
+	abis.push_back("arm64");
+	abis.push_back("x86_32");
 	abis.push_back("x86_64");
```

Comment:
I'm not sure this is correct to assume that `abis` == `env["arch"]` in this context.

The previous values were matching what's actually used by Android toolchains and if those are used as configuration values for gradle it might break exports.

CC @m4gr3d

Reply:
@akien-mga @aaronfranke This section is indeed incorrect as those values are passed to gradle build process which expects the Android supported abis.

Reply:
Worth a comment likely?

Reply:
@Zireael07 I've synced with @akien-mga and @aaronfranke and working on a fix for it.